### PR TITLE
chore: split the ci-check steps

### PR DESCRIPTION
**Pull request by abdonrd**
Split the ci-check steps. I expect that if we run it in parallel, we can save time.

Also looks better to find out what may be failing:

Screenshot 2021-11-24 at 14 11 21

### CODE REVIEW

File: .github/workflows/ci-check.yml [hunk 1]
```diff
@@ -10,24 +10,80 @@ concurrency:
...
-        uses: actions/setup-node@v2
+      - uses: actions/setup-node@v2
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --offline
+      - run: sudo apt-get install xvfb
+      - run: xvfb-run --auto-servernum yarn format:diff
```
Comment (abdonrd) [inline .github/workflows/ci-check.yml:29]: @jeffchew why the `xvfb-run --auto-servernum`? Can we remove it?

**Comment by abdonrd** [comment]
@jeffchew if we want to merge it, we also need to update the required checks.

File: .github/workflows/ci-check.yml (hunk 1 above)
Comment (jeffchew) [reply .github/workflows/ci-check.yml:29]: Not sure, this was originally put in by @asudoh .

Comment (abdonrd) [reply .github/workflows/ci-check.yml:29]: @asudoh could you confirm us? Thanks!

Comment (asudoh) [reply .github/workflows/ci-check.yml:29]: @abdonrd @jeffchew `xvfb-run` in this file is for ensuring GUI code (of browsers) runs in CI, notably for Karma, Puppeteer and Playwright. You can see the the code back in April for your reference: https://github.com/carbon-design-system/carbon-web-components/blob/v1.13.0/.github/workflows/ci-check.yml

Comment (abdonrd) [reply .github/workflows/ci-check.yml]: Ok, thanks Akira!

**Comment by abdonrd** [comment]
@jeffchew now it's ready!

**Comment by jeffchew** [comment]
Thanks @abdonrd , I just merged the Node 16 issue and now this one has conflicts, can you resolve?

**Comment by abdonrd** [comment]
@jeffchew done!

**Pull request merged**